### PR TITLE
cn0565 project (stm32 & aducm3029)

### PR DESCRIPTION
**Pull request by buha**
This is the initial version of the cn0565 project for Body
Impedance Analysis.

The projects currently builds for and runs on aducm3029 and SDP-K1.

`make` builds a custom application that communicates over UART to a
corresponding libiio app (sw_eit_ad5940).

`make TINYIIOD=y` builds an app that runs an iiod server. The same
libiio app (sw_eit_ad5940) may be used in iio mode to connect to
the iiod server, but iio-oscilloscope or other iio clients may be
used as well.

### CODE REVIEW

File: drivers/switch/adg2128/iio_adg2128.c [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#include <stdlib.h>
```
Comment (amiclaus) [inline drivers/switch/adg2128/iio_adg2128.c:1]: nit: missing header comment for the file

File: projects/cn0565/src/app/app.c [hunk 2]
```diff
@@ -0,0 +1,516 @@
+/*! ****************************************************************************
+ * @file:    eit.c
```
Comment (amiclaus) [inline projects/cn0565/src/app/app.c:2]: nit: header comment should be reworked.

File: projects/cn0565/src/app/parameters.h [hunk 3]
```diff
@@ -0,0 +1,81 @@
+/***************************************************************************//**
+ *   @file   ad9361/src/parameters.h
+ *   @brief  Parameters Definitions.
+ *   @author DBogdan (dragos.bogdan@analog.com)
```
Comment (amiclaus) [inline projects/cn0565/src/app/parameters.h:4]: nit: this needs an update too.

File: drivers/afe/ad5940/ad5940.h [hunk 4]
```diff
@@ -0,0 +1,4161 @@
...
+   ------------------------------------------------------------------------------------------------------------------------- */
+#define BITP_AGPIO_GP0CLR_CLR                 0            /*  Set the Output LOW */
+#define BITM_AGPIO_GP0CLR_CLR                0x000000FF    /*  Set the Output LOW */
+
+/* -------------------------------------------------------------------------------------------------------------------------
+          AGPIO_GP0TGL                         Pos/Masks         Description
+   ------------------------------------------------------------------------------------------------------------------------- */
+#define BITP_AGPIO_GP0TGL_TGL                 0            /*  Toggle the Output */
+#define BITM_AGPIO_GP0TGL_TGL                0x000000FF    /*  Toggle the Output */
+
```
Comment (amiclaus) [inline drivers/afe/ad5940/ad5940.h:163]: nit: looks like extra line

**Comment by buha** [comment]
Will track this in #1409

**Pull request closed without merging**